content/en/apps/guides/forms/versioning.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Versioning forms"
+linkTitle: "Versioning forms"
+weight: 
+description: >
+  Record the version of the form when creating reports
+relatedContent: >
+  apps/features/reports
+---
+
+_Added in cht-core 3.15.0 and cht-conf 3.10.0_
+
+When uploading app or contact xforms, cht-conf 3.10.0+ will automatically generate a version and include it in the form doc's
+`xmlVersion` property. The version has two properties.
+
+| Property | Description                                                                  |
+|----------|------------------------------------------------------------------------------|
+| `time`   | The time that the form was uploaded to the server in millis since the epoch. |
+| `sha256` | A hash of the xform content.                                                 |
+
+For example:
+
+```json
+  "xmlVersion": {
+    "time": 1658717177750,
+    "sha256": "6f0bbfe5a9a9ebeb25784165879afec5e311b197cbd76ade5698c83c22dd9a8f"
+  }
+```
+
+When a user fills in a form with an `xmlVersion` property, the version is copied in to the report doc as the `form_version`
```

Comment:
For absolute clarity, it could be helpful to include a [snippet of the JSON](https://github.com/medic/cht-core/issues/7561#issuecomment-1092200710) that gets added to the report doc.

Reply:
Added.